addons/mail/models/mail_thread.py

```diff
@@ -2403,6 +2403,15 @@ def _process_attachments_for_template_post(self, mail_template):
         """
         return {}
 
+    def _process_edit_label_for_post(self, body):
+        """Process the body to add a label to the edition.
+
+        :param body: the body of the message
+        :return: the body with the label added
+        """
+        edit_label = Markup("<span class='o-mail-message-edited'/>")
```

Comment:
When deleting a message (sending empty body), we should probably not have this tag.
Or if we really need, we should exclude this tag when checking "is empty".
This is failing a test.

Reply:
it's already checking if the body is empty or not in python. `msg_values = {
            'body': escape(body) + Markup("<span class='o-mail-message-edited'/>"),  # keep html if already Markup, otherwise escape
        }` The failing test was due to the mock problem, I will fix that